apps/extension/src/listeners/tabs-updated-prax-init.ts

```diff
@@ -3,14 +3,26 @@ import { PraxConnection } from '../message/prax';
 
 // trigger injected-connection-port to init when a known page is loaded.
 chrome.tabs.onUpdated.addListener(
-  (tabId, { status, discarded }, { url }) =>
+  (tabId, { status, discarded, url: changeInfoUrl }, { url: tabUrl }) => {
+    // Unfortunately, no context for tab
+    // void chrome.runtime.getContexts({}).then(console.log);
```

Comment:
@turbocrime, sadly the runtime is only reporting BACKGROUND context here. This means all of the code below is not executing as we expect.
<img width="613" alt="Screenshot 2024-06-20 at 1 21 43 PM" src="https://github.com/prax-wallet/web/assets/16624263/80bf8f61-8beb-4c8d-9860-7b15b577f58b">